EventFilter/L1TXRawToDigi/plugins/L1TTwinMuxRawToDigi.cc

```diff
@@ -10,6 +10,14 @@
 //   C. F. Bedoya  - CIEMAT
 //   G. Codispoti -- INFN Bologna
 //   J. Pazzini   -- INFN Padova
+//   C. Heidemann -- RWTH Aachen
+//
+//
+//   Changes & Notes:
+//   This version is able to unpack the normal and the zero suppressed data
+//   For storing the additional info from the ud flag in the raw data and not changing the output data format of the unpacker, it will be stored as the sign to the station number for Phi In and Out segments. ud == 0 is stored as a positive station number, while ud==1 is stored as a negative station number. Use abs(station) to obtain the real station. 
```

Comment:
This comment is not true, i.e. station is not being moved to negative values for ud=1, right? 
Same applies to the 94X version of this PR: #20554

Reply:
You are right. The comment is wrong. It will be fixed in a future version.

Reply:
@carstenheidemann
please address comments so we can re-test this PR.